src/main/scala/com/reagroup/appliedscala/urls/savemovie/NewMovieRequest.scala

```diff
@@ -18,4 +18,10 @@ object NewMovieRequest {
     * Hint: The keys in the JSON are named exactly the same as the fields in `NewMovieRequest`.
     */
 
+  implicit val newMovieRequestDecoder: Decoder[NewMovieRequest] = cursor => {
```

Comment:
Answer key did it like this:
`  implicit val decoder: Decoder[NewMovieRequest] = deriveDecoder
`

i guess that was what the hint wanted me to do: 
`    * Hint: The keys in the JSON are named exactly the same as the fields in `NewMovieRequest`.
`